src/listeners.js

```diff
@@ -0,0 +1,17 @@
+import { displayWeather } from './ui';
+
+function loadListeners() {
+  document.querySelector('.search-submit').addEventListener('click', () => {
+    const place = document.querySelector('.search').value;
+    const currentPlace = document
+      .querySelector('.place')
+      .textContent.split(',')[0];
+    if (place !== '' && place !== currentPlace) displayWeather(place);
```

Comment:
if(place && place !== currentPlace).
Also, add curly brackets, it separates blocks and adds readability